crates/typst/src/export/render.rs

```diff
@@ -400,9 +406,16 @@ fn render_outline_glyph(
 
         // Premultiply the text color.
         let Paint::Solid(color) = text.fill;
-        let c = color.to_rgba();
-        let color =
-            bytemuck::cast(sk::ColorU8::from_rgba(c.r, c.g, c.b, 255).premultiply());
+        let [r, g, b, _] = color.to_rgba().to_vec4();
+        let color = bytemuck::cast(
+            sk::ColorU8::from_rgba(
+                (r * 255.0).round() as u8,
+                (g * 255.0).round() as u8,
+                (b * 255.0).round() as u8,
+                255,
+            )
```

Comment:
Since we've had this before, perhaps extract it? Or make a `From` impl. Or go through `sk::Color`'s existing from impl.